test/index.ts

```diff
@@ -29,6 +29,12 @@ const dotnetIsolated7 = {
   response: "Welcome to Azure Functions! .NET 7 Preview 5"
 }
 
+const dotnetIsolated8 = {
+  package: `${storagePath}/dotnet-isolated8-functions.zip`,
+  invoke: "/api/DotnetIsolated8HttpFunction",
```

Comment:
Does this exist? If not we need to create one

Reply:
Yes, I created and uploaded them (The tests are using this)